flask/test/auth_tests/test_auth.py

```diff
@@ -0,0 +1,78 @@
+import pytest
+from auth import get_user_token_string
+
+from boxwise_flask.auth_helper import (  # ,; decode_jwt,; requires_auth,; 
+    user_can_access_base,
+    authorization_test,
+    get_rsa_key,
+    get_token_from_auth_header,
+    AuthError,
+    requires_auth,
+    decode_jwt
+)
+
+def test_get_valid_jwt():
+    token = get_token_from_auth_header(get_user_token_string())
+    key = get_rsa_key(token)
+    assert key != None
+
+def test_get_invalid_jwt_no_auth_header():
+    with pytest.raises(AuthError):
```

Comment:
It would be nice if you could assert the error messages of the different AuthErrors

Reply:
@mc18g13 still need an answer here